libs/ledgerjs/packages/types-live/src/bridge.ts

```diff
@@ -94,7 +95,9 @@ export interface CurrencyBridge {
     syncConfig: SyncConfig;
     preferredNewAccountScheme?: DerivationMode;
   }): Observable<ScanAccountEvent>;
-  getPreloadStrategy?: (currency: CryptoCurrency) => PreloadStrategy;
+  // Optional logic to apply to complete missing data,
+  // when importing accounts from live-desktop to live-mobile
+  importAccountData?: (accountData: AccountData) => Partial<Account>;
```

Comment:
`Partial<Account>` is not something we accept anywhere so i think it's dangerous to define as this. i would more see that we need some sort of `Account -> A` where A would be the Account of the family. not something that lives yet in the accountbridge, but we could introduce a new parametric type. i can give a try to see if things could work like this, but ultimately we want our sync() to no longer take a Account but actually a A. that way, we stop doing ifs everywhere on the presence or not of the "resources", we would know it settled.

🤔  (just thinking out loud)